domain/infosync/info.go

```diff
@@ -1228,6 +1230,10 @@ func ConfigureTiFlashPDForTable(id int64, count uint64, locationLabels *[]string
 
 // ConfigureTiFlashPDForPartitions configures pd rule for all partition in partitioned tables.
 func ConfigureTiFlashPDForPartitions(accel bool, definitions *[]model.PartitionDefinition, count uint64, locationLabels *[]string, tableID int64) error {
+	if config.GetGlobalKeyspaceName() != "" && count > 0 {
+		// TiDB 6.6 doesn't support tiflash multi-tenancy yet.
+		return errors.Trace(dbterror.ErrNotSupportedYet.GenWithStackByArgs("TiFlash replica doesn't support API V2"))
```

Comment:
Ditto.

Reply:
This check has been done in SetPlacementRules, removed in latest commit.